README.md

```diff
@@ -15,12 +15,40 @@ To run this sample you will need the following:
 
 ## Register the sample
 
-Create a new app at [apps.dev.microsoft.com](https://apps.dev.microsoft.com), or follow these [detailed steps](https://azure.microsoft.com/en-us/documentation/articles/active-directory-v2-app-registration/).  Make sure to:
+1. Sign in to the [Azure portal](https://portal.azure.com/) using either a work or school account, or a personal Microsoft account.
 
-- Copy down the **Application Id** assigned to your app, you'll need it soon.
-- Add the **Web** platform for your app.
-- Enter the correct **Redirect URI**. The redirect uri indicates to Azure AD where authentication responses should be directed - the default for this sample is `http://localhost:3000/auth/openid/return'`.
-- Add a new **Application secret** by clicking the **Generate new password** button. This value will not be displayed again, so save the result in a temporary location as you'll need it in the next step.
+1. If your account is present in more than one Azure AD tenant:
+    - Select your profile from the menu on the top right corner of the page, and then **Switch directory**.
+    - Change your session to the Azure AD tenant where you want to create your application.
+
+1. Navigate to [Azure Active Directory > App registrations (Preview)](https://go.microsoft.com/fwlink/?linkid=2083908) to register your app.
+
+1. Select **New registration.**
+
+1. When the **Register an application** page appears, enter your app's registration information:
+    - In the **Name** section, enter a meaningful name that will be displayed to users of the app. For example: MyWebApp
+    - In the **Supported account types** section, select **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**.
+
+    If there are more than one redirect URIs, you'll need to add these from the **Authentication** tab later after the app has been successfully created.
+
+1. Select **Register** to create the app.
+
+1. On the app's **Overview** page, find the **Application (client) ID** value and record it for later. You'll need this value to configure the application later in this project.
+
+1. In the list of pages for the app, select **Authentication**.
+    - In the **Redirect URIs** section, select **Web** in the combo-box and enter the following redirect URI:
+    http://localhost:3000/auth/openid/return
```

Comment:
you might want to use back quotes around the two URIs.

Reply:
Thank you for the review. Updated.
I have the permission to merge it.